src/test/java/com/openlattice/conductor/ConductorBootstrap.java

```diff
@@ -30,29 +30,26 @@
  * @author Matthew Tamayo-Rios &lt;matthew@openlattice.com&gt;
  */
 public class ConductorBootstrap {
+
     protected static final Linker LINKER;
 
     static {
-        LINKER = new Linker();
         final var logger = LoggerFactory.getLogger( ConductorBootstrap.class );
+        LINKER = new Linker();
+        String[] profiles = new String[3];
+        profiles[0] = "postgres";
+        profiles[1] = "keras";
         if ( NetworkUtils.isRunningOnHost( "bamboo.openlattice.com" ) ) {
-            LoggerFactory.getLogger( ConductorBootstrap.class ).info( "Running on bamboo!" );
-            try {
-                LINKER.start( "awstest", "postgres", "keras" );
-            } catch ( Exception e ) {
-                logger.error( "Unable to bootstrap condcutor with profiles: {}",
-                        LINKER.getContext().getEnvironment().getActiveProfiles() );
-                throw new IllegalStateException( "Unable to to boostrap conductor");
-            }
+            logger.info( "Running on bamboo!" );
+            profiles[2] = "awstest";
```

Comment:
Not important, but can we use predefined constants here, where it's available? For example `ConfigurationConstants.Profiles` or `PostgresPod.PROFILE`

Reply:
👍